.env.test

```diff
@@ -0,0 +1,6 @@
+DB_PASSWORD=password
```

Comment:
Not related to this file, but you have 2 `.DS_Store` files here which should not have been committed. Also, given that this file (`.env.test` im talking about here) is in your `.gitignore` why is it included? Most likely it was committed before it was added to the `.gitignore` file. Either way they should be removed